wiki/History_of_osu!/osu!_wiki/es.md

```diff
@@ -0,0 +1,37 @@
+# Historia de la osu! wiki
+
+## MediaWiki (2011 - 2017)
+
+![](img/ow_MW.jpg "Captura de pantalla completa de la osu! wiki funcionando en MediaWiki, tomada en 17/2/2017")
+
+| Fecha, Hora | Descripción |
+| :-- | :-- |
+| 5/12/2011 T08:13Z | Primera edición de la osu! wiki por Ephemeral. |
```

Comment:
```suggestion
| 5/12/2011 (08:13 UTC) | Primera edición de la osu! wiki por Ephemeral. |
```

Reply:
this change can be made in the english article as well